### PR TITLE
Fix POST DisaggregationType #928

### PR DESCRIPTION
**Pull request by Menda**
Purpose
=====

Fixes POST requests to /api/disaggregationtype/

Ticket
====

https://github.com/toladata/TolaActivity/issues/928

### CODE REVIEW

File: feed/serializers.py [hunk 1]
```diff
@@ -225,6 +225,7 @@ class Meta:
 
 class DisaggregationTypeSerializer(serializers.HyperlinkedModelSerializer):
     id = serializers.ReadOnlyField()
+    disaggregation_type = serializers.CharField(max_length=135)
```
Comment (jefmoura) [inline feed/serializers.py]: We already have '\_\_all__' in the Meta, so you don't need to add the `disaggregation_type` field here.

File: feed/tests/test_disaggregationtypeview.py [hunk 2]
```diff
@@ -35,11 +36,53 @@ def test_list_disaggregationtype_normaluser(self):
...
         self.assertEqual(len(response.data), 1)
+
+
+class DisaggregationTypeCreateViewsTest(TestCase):
+    def setUp(self):
+        self.organization = factories.Organization()
+        self.factory = APIRequestFactory()
+
+    def test_create_disaggregationtype_normaluser_missing_values(self):
+        user = factories.User(is_superuser=True, is_staff=True)
```
Comment (jefmoura) [inline feed/tests/test_disaggregationtypeview.py]: `superuser` isn't `normaluser`. If it's supposed to use a superuser in the test, just change the name of the test.

Comment (jefmoura) [reply feed/tests/test_disaggregationtypeview.py]: The other test is called normaluser as well but it uses a superuser

File: feed/serializers.py (hunk 1 above)
Comment (Menda) [reply feed/serializers.py]: This override is on purpose. I need to make this field mandatory, because in the model it's not.

Comment (jefmoura) [reply feed/serializers.py]: Ok

**Pull request merged**